docs/user/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -16,21 +16,21 @@ Trouble with Kyma Eventing may be caused by various issues, so this document gui
 
 Follow these steps to detect the source of the problem:
 
-### Step 1: Check the status of the EventingBackend CR
+### Step 1: Check the status of the Eventing CR
```

Comment:
When you first mention 'custom resource', use the full form (use lowercase for that) and the abbreviation in brackets (CR). Then, you can use the abbreviation throughout the doc.